pkg/controller/mutators/stats_reporter.go

```diff
@@ -101,33 +102,29 @@ func NewStatsReporter() StatsReporter {
 // required for this request to complete.  The outcome of the ingestion attempt is recorded via the
 // status argument.
 func (r *reporter) ReportMutatorIngestionRequest(ms MutatorIngestionStatus, d time.Duration) error {
-	ctx, err := tag.New(
-		context.Background(),
-		tag.Insert(mutatorStatusKey, string(ms)),
-	)
-	if err != nil {
-		return err
-	}
-
-	return metrics.Record(ctx, responseTimeInSecM.M(d.Seconds()))
+	responseTimeInSecM.Record(context.Background(), d.Seconds(), metric.WithAttributes(attribute.String(statusKey, string(ms))))
+	mutatorIngestionCountM.Add(context.Background(), 1, metric.WithAttributes(attribute.String(statusKey, string(ms))))
+	return nil
 }
 
 // ReportMutatorsStatus reports the number of mutators of a specific status that are present in the
 // mutation System.
-func (r *reporter) ReportMutatorsStatus(ms MutatorIngestionStatus, n int) error {
```

Comment:
As per elsewhere, `report` functions should still exist. They should update some internal status.